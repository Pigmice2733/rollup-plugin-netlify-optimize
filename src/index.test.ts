import { createRollupConfig } from './index'

test('creates a config', () => {
  expect(
    createRollupConfig({ babelConfig: {}, routes: {} }, 'production'),
  ).toMatchSnapshot()
})
