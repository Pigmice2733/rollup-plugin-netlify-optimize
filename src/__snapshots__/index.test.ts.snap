// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a config 1`] = `
Object {
  "experimentalCodeSplitting": true,
  "input": Array [
    "src/index.tsx",
  ],
  "output": Object {
    "dir": "dist",
    "format": "es",
  },
  "plugins": Array [
    Object {
      "load": [Function],
      "name": "postcss",
      "onwrite": [Function],
      "resolveId": [Function],
      "transform": [Function],
    },
    Object {
      "load": [Function],
      "name": "babel",
      "options": [Function],
      "resolveId": [Function],
      "transform": [Function],
    },
    Object {
      "name": "node-resolve",
      "onwrite": [Function],
      "options": [Function],
      "resolveId": [Function],
    },
    Object {
      "generateBundle": [Function],
      "name": "html",
      "options": [Function],
    },
    Object {
      "name": "terser",
      "transformBundle": [Function],
    },
  ],
}
`;
